src/screens/TopicPageScreen/index.js

```diff
@@ -109,6 +108,12 @@ const TopicPageScreen = (props) => {
     extrapolate: 'clamp'
   });
 
+  const opacityImage = scrollY.interpolate({
+    inputRange: [0, 100],
+    outputRange: [0, 1],
+    extrapolate: 'clamp'
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of `opacityImage` is a good addition if you want to control the opacity of an image based on the scroll position. Just ensure that the input range `[0, 100]` and output range `[0, 1]` are suitable for your use case.
```javascript
const opacityImage = scrollY.interpolate({
  inputRange: [0, 100],
  outputRange: [0, 1],
  extrapolate: 'clamp'
});
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->